TACC/Core-CMS
# Quick: Remove Unused JavaScript

## Overview

Remove cruft JavaScript files.

## Related

none

## Changes

- Remove `site.js` and call to load it.
- Remove `nav_portal.js` (long since deprecated and never loaded).

## Testing

1. Confirm that this console log does not appear:
    ```js
    Core \`site.js\` has no logic.
     ```
2. Confirm that `site.js` is not loaded.

## Screenshots

![No Console Log](https://user-images.githubusercontent.com/62723358/162064972-da00fdcd-6e06-4563-b730-0d73df3168fe.png)

![No Site js Load](https://user-images.githubusercontent.com/62723358/162064975-4351583b-659c-4bba-b031-8abd2172d8b3.png)
